runtime/doc/quickfix.txt

```diff
@@ -1551,34 +1574,6 @@ the :set command.  There are two backslashes in front of a comma, one for the
 :set command and one to avoid recognizing the comma as a separator of error
 formats.
 
-
-Filtering messages
```

Comment:
This section was not deleted in Vim. Only `*cfilter-plugin*` was moved.

Reply:
![Screenshot from 2020-09-21 11-16-50](https://user-images.githubusercontent.com/6953273/93745330-ff7d8280-fbfb-11ea-9b6a-0113ceb1dbc6.png)

Reply:
What do you mean? If you look at the screen shot of the commit in vim, you will see that there is no text between the line `The backslah before each space is required for the set command.` and  `==============================`

If you want I can just add the text there so it will become
```
The backslash before each space is
required for the set command.

Filtering messages

=============================================================================
8. The directory stack				*quickfix-directory-stack*
```

Is that ok?

Reply:
Done